docs/learn/learn-architecture.md

```diff
@@ -47,11 +47,11 @@ like smart contracts, privacy, or scalability &mdash; still, others might be exp
 architectures that are not necessarily blockchain in nature.
 
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} provides many ways to secure a slot
-for a parachain for a particular length of time. Parathreads are part of a pool that shares slots
-and must-win auctions for individual blocks. Parathreads and parachains have the same API; their
-difference is economic. Parachains will have to reserve
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} for the duration of their slot lease;
-parathreads will pay on a per-block basis. Parathreads can become parachains, and vice-versa.
+for a parachain for a particular length of time. On-demand parachains share slots with other
+parachains. Permanent and on-demand parachains have the same API; their difference is economic.
```

Comment:
There is no guarantee they are going to be permanent. Even the lease currently can be for a macximum duration of 2 years. We can refer to the parachains with bulk coretime as "regular" chanis or just parachains.

```suggestion
parachains. Both regular and on-demand parachains have the same API; their difference is economic.
```